### ISSUE TITLE
nil pointer dereference in vochain/appsetup.go:65

### ISSUE THREAD

**Issue by p4u** [vocdoni/vocdoni-node#1249]
```
2024-02-09T23:41:31.748Z DBG apirest/apirest.go:197 > request: {"txPayload":"Cu0BEuoBCAEQDhrjARIUgs1vznhqYBNpSC6C6QBSs1qkQQEqIGgPPhyQUHntlAHUIpKahswv1MUgU3eE8t/kgiCD1sHVMkJpcGZzOi8vYmFmeWJlaWU0cnNtNHNuNXlwaGdyZXV6ZGRmc2ZycWI1ZHlub3ZiM3JneHo1cmNvaWZqN3Q3NmRsa3VgAXoAggECCAGaAQgIARADIAMoAaABENIBQmlwZnM6Ly9iYWZ5YmVpY2J0bGt1eng1ZWdpaXBiYmw2YnpncGd1ZXk1ZmVlazVwcmtheHN3YXlyb3RueTV5eHZtceABBpACAKACgPUkEkG+0OTfqzX/KEVu8H3/V+c7B+SGh7EyRvOjbE62ifjJ0QHx6+2dEP3tmIsT448THIhGGtGfo5oTGBkFnJRhAgz7AQ==","metadata":"eyJ0aXRsZSI6eyJkZWZhdWx0IjoiSG93IGRvIHlvdSB0YWtlIHlvdXIgS2l3aT8ifSwidmVyc2lvbiI6IjEuMCIsImRlc2NyaXB0aW9uIjp7ImRlZmF1bHQiOiIifSwibWVkaWEiOnt9LCJxdWVzdGlvbnMiOlt7ImNob2ljZXMiOlt7InRpdGxlIjp7ImRlZmF1bHQiOiJTa2luIG9uIn0sInZhbHVlIjoxfSx7InRpdGxlIjp7ImRlZmF1bHQiOiJTa2luIG9mZiJ9LCJ2YWx1ZSI6Mn0seyJ0aXRsZSI6eyJkZWZhdWx0IjoiSSBkb24ndCBsaWtlIEtpd2kifSwidmFsdWUiOjN9XSwiZGVzY3JpcHRpb24iOnsiZGVmYXVsdCI6IiJ9LCJ0aXRsZSI6eyJkZWZhdWx0IjoiSG93IGRvIHlvdSB0YWtlIHlvdXIgS2l3aT8ifX1dLCJ0eXBlIjp7Im5hbWUiOiJzaW5nbGUtY2hvaWNlLW11bHRpcXVlc3Rpb24iLCJwcm9wZXJ0aWVzIjpudWxsfX0=","txHash":"","electionID":"","m...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x329cff3]

goroutine 181067818 [running]:
go.vocdoni.io/dvote/vochain.(*BaseApplication).SetDefaultMethods.func4({0xc00038cb40?, 0x133, 0x3b?})
	go.vocdoni.io/dvote/vochain/appsetup.go:65 +0x53
go.vocdoni.io/dvote/vochain.(*BaseApplication).SendTx(0x24?, {0xc00038cb40?, 0x24?, 0x19d?})
	go.vocdoni.io/dvote/vochain/app.go:341 +0x2a
go.vocdoni.io/dvote/api.(*API).sendTx(0xc019478840?, {0xc00038cb40?, 0xc00a8220a0?, 0x42?})
	go.vocdoni.io/dvote/api/helpers.go:40 +0x2b
go.vocdoni.io/dvote/api.(*API).electionCreateHandler(0xc012676370, 0xc01ddaf050, 0x4550500?)
	go.vocdoni.io/dvote/api/elections.go:496 +0x2c8
go.vocdoni.io/dvote/httprouter/apirest.(*API).RegisterMethod.func1({{0x3633dc0, 0xc01ddaf050}, {0xc169cbeaec9b7dc8, 0x13ace82d5664, 0x603d7a0}, {0xc01ddaf0c0, 0x2, 0x2}, 0xc01ddaf080})
	go.vocdoni.io/dvote/httprouter/apirest/apirest.go:231 +0x6b
created by go.vocdoni.io/dvote/httprouter.(*HTTProuter).AddPublicHandler.(*HTTProuter).routerHandler.func1 in goroutine 181067768
	go.vocdoni.io/dvote/httprouter/httprouter.go:285 +0x3c5
vocdoni version "8e1ec02"
```

**Comment by altergui** [comment]
i also came across this, in a random testsuite run

```
gateway0-1  | 2024-02-19T20:45:03.291Z DBG faucet/faucet.go:57 > faucet request amount=100000 from=0x1F49F5418218044a24B96eD2D36bE37461D9Dcd0
gateway0-1  | 2024-02-19T20:45:03.293Z DBG httprouter/message.go:76 > http response msg="{\"amount\":\"100000\",\"faucetPackage\":\"eyJmYXVjZXRQYXlsb2FkIjoiQ1AzVjZ0TURFaFFmU2ZWQmdoZ0VTaVM1YnRMVGErTjBZZG5jMEJpZ2pRWT0iLCJzaWduYXR1cmUiOiJEaGp4RmRubUx0UGprVWVGR2xSYkxvRXh5aEZESUVQa0l0clFOc0FVQVBNVUtTZUs4SW5WQy96T3VwWE5EbloyMzdRMzgrSnAzajFzelBsT0NwMUpWZ0E9In0=\"}" status=200
gateway0-1  | 2024-02-19T20:45:03.294Z DBG middleware/logger.go:159 > "GET http://gateway0:9090/v2/open/claim/0x1F49F5418218044a24B96eD2D36bE37461D9Dcd0 HTTP/1.1" from 172.28.0.8:37924 - 200 252B in 2.929757ms
gateway0-1  | 2024-02-19T20:45:03.299Z DBG apirest/apirest.go:197 > request: {"txPayload":"CuoBSucBCBcQABpCaXBmczovL2JhZnliZWlhbW1vY3htYTVtcXd4cnVvZ25heG51NmE1a2d4MnI0bW1ldmdycnNncGN5dDdvdWR5ZmltIhQfSfVBghgESiS5btLTa+N0Ydnc0CplCiAI/dXq0wMSFB9J9UGCGARKJLlu0tNr43Rh2dzQGKCNBhJBDhjxFdnmLtPjkUeFGlRbLoExyhFDIEPkItrQNsAUAPMUKSeK8InVC/zOupXNDnZ237Q38+Jp3j1szPlOCp1JVgA6ILfxZJ+10Q2pSeqqCt5jO4I0yd9uofi5yT7RMyulZhAtEkHZh2AYOY3uAYpNpxP2HI9Vi7ab9TYe7qziGrlYWWG7wlvYPqlM4qmc9WDHLFHbuwIhQzHwYKP6vhgkZzwehizxAQ==","metadata":"eyJ2ZXJzaW9uIjoiMS4wIiwibmFtZSI6eyJkZWZhdWx0IjoidGVzdCBhY2NvdW50IDB4MUY0OUY1NDE4MjE4MDQ0YTI0Qjk2ZUQyRDM2YkUzNzQ2MUQ5RGNkMCJ9LCJkZXNjcmlwdGlvbiI6eyJkZWZhdWx0IjoidGVzdCBkZXNjcmlwdGlvbiJ9fQ==","txHash":"","metadataURL":""}
gateway0-1  | panic: runtime error: invalid memory address or nil pointer dereference
gateway0-1  | [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x32a0db3]
gateway0-1  | 
gateway0-1  | goroutine 3381 [running]:
gateway0-1  | go.vocdoni.io/dvote/vochain.(*BaseApplication).SetDefaultMethods.func4({0xc008059040?, 0x130, 0x3b?})
gateway0-1  | 	go.vocdoni.io/dvote/vochain/appsetup.go:65 +0x53
gateway0-1  | go.vocdoni.io/dvote/vochain.(*BaseApplication).SendTx(0x24?, {0xc008059040?, 0x24?, 0x8b?})
gateway0-1  | 	go.vocdoni.io/dvote/vochain/app.go:350 +0x2a
gateway0-1  | go.vocdoni.io/dvote/api.(*API).sendTx(0xc008714ec0?, {0xc008059040?, 0xc007e57b30?, 0x42?})
gateway0-1  | 	go.vocdoni.io/dvote/api/helpers.go:40 +0x2b
gateway0-1  | go.vocdoni.io/dvote/api.(*API).accountSetHandler(0xc001c6e5f0, 0xc0087509c0, 0x10000c008119968?)
gateway0-1  | 	go.vocdoni.io/dvote/api/accounts.go:289 +0x2b8
gateway0-1  | go.vocdoni.io/dvote/httprouter/apirest.(*API).RegisterMethod.func1({{0x3639000, 0xc0087509c0}, {0xc16d0d53d1d7f225, 0x2f5be694f, 0x60457c0}, {0xc008750a30, 0x2, 0x2}, 0xc0087509f0})
gateway0-1  | 	go.vocdoni.io/dvote/httprouter/apirest/apirest.go:231 +0x6b
gateway0-1  | created by go.vocdoni.io/dvote/httprouter.(*HTTProuter).AddPublicHandler.(*HTTProuter).routerHandler.func1 in goroutine 3363
gateway0-1  | 	go.vocdoni.io/dvote/httprouter/httprouter.go:285 +0x3c5
gateway0-1 exited with code 2
```
(line numbers might be slightly offset since i was working on my dirty branch)

this was triggered by end2endtest, here's the log of the test:
```
2024-02-19T20:45:03.286Z INF end2endtest/main.go:169 > starting setup test=plaintextelection
2024-02-19T20:45:03.287Z DBG apiclient/client.go:169 > http request body=null path=/v2/accounts/0x1F49F5418218044a24B96eD2D36bE37461D9Dcd0 type=GET
2024-02-19T20:45:03.295Z INF end2endtest/helpers.go:111 > creating Vocdoni account 0x1F49F5418218044a24B96eD2D36bE37461D9Dcd0
2024-02-19T20:45:03.297Z DBG apiclient/client.go:169 > http request body={"metadata":"eyJ2ZXJzaW9uIjoiMS4wIiwibmFtZSI6eyJkZWZhdWx0IjoidGVzdCBhY2NvdW50IDB4MUY0OUY1NDE4MjE4MDQ0YTI0Qjk2ZUQyRDM2YkUzNzQ2MUQ5RGNkMCJ9LCJkZXNjcmlwdGlvbiI6eyJkZWZhdWx0IjoidGVzdCBkZXNjcmlwdGlvbiJ9fQ==","metadataURL":"","txHash":"","txPayload":"CuoBSucBCBcQABpCaXBmczovL2JhZnliZWlhbW1vY3htYTVtcXd4cnVvZ25heG51NmE1a2d4MnI0bW1ldmdycnNncGN5dDdvdWR5ZmltIhQfSfVBghgESiS5btLTa+N0Ydnc0CplCiAI/dXq0wMSFB9J9UGCGARKJLlu0tNr43Rh2dzQGKCNBhJBDhjxFdnmLtPjkUeFGlRbLoExyhFDIEPkItrQNsAUAPMUKSeK8InVC/zOupXNDnZ237Q38+Jp3j1szPlOCp1JVgA6ILfxZJ+10Q2pSeqqCt5jO4I0yd9uofi5yT7RMyulZhAtEkHZh2AYOY3uAYpNpxP2HI9Vi7ab9TYe7qziGrlYWWG7wlvYPqlM4qmc9WDHLFHbuwIhQzHwYKP6vhgkZzwehizxAQ=="} path=/v2/accounts type=POST
2024-02-19T20:45:03.336Z FTL end2endtest/main.go:175 > Post "http://gateway0:9090/v2/accounts": EOF
goroutine 11 [running]:
runtime/debug.Stack()
	runtime/debug/stack.go:24 +0x5e
go.vocdoni.io/dvote/log.Fatal({0xc000e41e90, 0x1, 0x1})
	go.vocdoni.io/dvote/log/log.go:212 +0x65
main.setupAndRun({0x3a8daf8, {0x394aeba, 0x63}, {0xc000705840, 0x38}}, 0x0?, 0xc000d496b0)
	go.vocdoni.io/dvote/cmd/end2endtest/main.go:175 +0x175
main.main.func2({0x3a8daf8, {0x394aeba, 0x63}, {0xc000705840, 0x38}}, 0x0?, 0x0?)
	go.vocdoni.io/dvote/cmd/end2endtest/main.go:245 +0xfb
created by main.main in goroutine 1
	go.vocdoni.io/dvote/cmd/end2endtest/main.go:267 +0x285
```

**Comment by altergui** [comment]
@p4u fixed this in 85b22b03d0b3369470606783ffa819293fa70de4
